test/vike-node/server/index-express.ts

```diff
@@ -14,8 +14,7 @@ new Worker(new URL('./worker.mjs', import.meta.url))
 async function startServer() {
   await init()
   const app = express()
-  app.use(express.text()) // Parse & make HTTP request body available at `req.body`
-  app.all('/_telefunc', async (req, res) => {
+  app.all('/_telefunc', express.text(), async (req, res) => {
```

Comment:
It's a refacto

Reply:
Usually it's best to keep refactors outside PRs as it makes it harder for the reader to review the PR. One thing we can do is to directly push refactor commits (no need to create a PR, or a self-merging PR if confidence is lower) before opening the PR. (That's actually something I do a lot internally with Vike core even though no ones reviewing my work, because it makes it easier for me to re-review my changes, and also easier to revert/dig into potential issues I overlooked.)